### DISCUSSION TITLE
graphql - orderBy options

### DISCUSSION THREAD

**Discussion by giga-sean** [snapshot-labs/snapshot#2935] [Q&A] [answered]
what the possible strings you can pass into the orderBy parameter here?

query {
  spaces(
    first: 20,
    skip: 0,
    orderBy: "created",
    orderDirection: asc
  ) {
    id
    name
    about
    network
    symbol
    strategies {
      name
      params
    }
    admins
    members
    filters {
      minScore
      onlyMembers
    }
    plugins
  }
}

**Comment by metrox-eth** [comment, marked as answer]
It should work with 'created', 'start' or 'end'.

**Comment by Saberhabeelshohdy** [comment]
**lol **

**Comment by Saberhabeelshohdy** [comment]
- [ ] mim
